src/converter/one/proto/Proto.java

```diff
@@ -63,27 +66,45 @@ public Proto field(int index, byte[] bytes) {
         return this;
     }
 
+    public Proto field(int index, ByteBuffer bytes) {
```

Comment:
Is this method used?

Reply:
Leftover from a previous iteration, fixed in 2359e43708a736b01714e3842adfd6a91ad6e203